src/network/services/auth/index.ts

```diff
@@ -129,3 +129,31 @@ export const verifyOtp = (values: object) => {
     }
   };
 };
+
+export const changePassword = (values: object) => {
+  return async (dispatch) => {
+  
+    try {
+      let state = store.getState();
+      const { auth } = state;
+      const res = await NetworkCall.fetch(
+        UserRequest.changePassword(values, auth.loggedInUser.token)
+        //UserRequest.changePassword(values, auth.token)
+      );
+      return res;
+    } catch (errors) {
+      console.log(errors.error.data);
```

Comment:
Do we need this? If yes, can we use `console.error()`?

Reply:
Removed